src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Classes/WindowsSetting.cs

```diff
@@ -62,5 +63,19 @@ public WindowsSetting()
         /// Gets or sets the Windows build since this settings is not longer present.
         /// </summary>
         public uint? DeprecatedInBuild { get; set; }
+
+        /// <summary>
+        /// Gets or sets the the value with the generated area path as string.
+        /// This Property IS NOT PART OF THE DATA IN "WindowsSettings.json".
+        /// This property will be filled on runtime by "WindowsSettingsPathHelper".
+        /// </summary>
+        public string? JoinedAreaPath { get; set; }
+
+        /// <summary>
+        /// Gets or sets the the value with the generated full settings path (App and areas) as string.
+        /// This Property IS NOT PART OF THE DATA IN "WindowsSettings.json".
+        /// This property will be filled on runtime by "WindowsSettingsPathHelper".
+        /// </summary>
+        public string? JoinedFullSettingsPath { get; set; }
```

Comment:
It doesn't show me the variable contents. Do you have any idea what I did wrong?

Reply:
> It doesn't show me the variable contents. Do you have any idea what I did wrong?

No it works. But don't ask me why.

Reply:
@davidegiacometti 
For your info: The missing values of json will be initialized with the init values defined in the class.
![image](https://user-images.githubusercontent.com/61519853/131216655-aae91331-1334-4791-9666-f684cb0f675b.png)

Reply:
> Replace `string?` with `string` for `JoinedAreaPath` and `JoinedFullSettingsPath` since `string` can be null.

This is not allowed. As I read because it is a base class.